### DISCUSSION TITLE
ISO to Month, day year format

### DISCUSSION THREAD

**Discussion by sfonua10** [js-fns/date-fns#3148] [General]
How can I change this `2022-08-06T00:00-06:00` into this `Aug 06, 2022`. 

Here's my code:
```
import parseISO from "date-fns/parseISO";

parseISO(blogs[0].fields.date)
```

But it's output i s this: `Sat Aug 06 2022 00:00:00 GMT-0600 (Mountain Daylight Time)`

I want this `Aug 06, 2022`

Can someone help?

**Comment by fturmel** [comment]
You were half-way there. After parsing the string to a Date instance, you need to format it back to a string. You can use the `format` function for this and use the appropriate tokens to get the desired output.

https://date-fns.org/docs/format

```js
import { parseISO, format } from "date-fns";

const date = parseISO("2022-08-06T00:00-06:00");

// option 1: localizable token `P`
const str1 = format(date, "PP", new Date());
console.log(str1); // Aug 6, 2022

// option 2: manually define each token
const str2 = format(date, "MMM dd, yyyy", new Date());
console.log(str2); // Aug 06, 2022
```
